cryptarchia/cryptarchia.py

```diff
@@ -143,112 +161,68 @@ def encode_sk(self) -> bytes:
     def encode_pk(self) -> bytes:
         return int.to_bytes(self.pk, length=32, byteorder="big")
 
-    def evolve(self) -> "Coin":
-        h = blake2b(digest_size=32)
-        h.update(b"coin-evolve")
-        h.update(self.encode_sk())
-        h.update(self.nonce)
-        evolved_nonce = h.digest()
-
-        return Coin(nonce=evolved_nonce, sk=self.sk, value=self.value)
-
-    def commitment(self) -> Id:
-        # TODO: mocked until CL is understood
```

Comment:
Is it safe to remove this TODO?

Reply:
Yes, this is the correct commitment implementation now